nss-day-cohort-43-csharp/MVC-Orange-Iguanas
# Jt tabloid View Post's comments

Now have the ability to look at all comments on each post.
To test:
1.) Click on Posts in nav bar
2.) Click on the eyebal icon (view) on any post
3.) On the post detail page, you should now see a comment icon below "written by" and to the right of the trash icon
4.) click on the comment icon, and you should see all the comments for that post


In order to test, you may need to run the following to create the following SQL query to create comments:
INSERT INTO  Comment (
                            PostId, UserProfileId, Subject, Content, CreateDateTime)
                        OUTPUT INSERTED.ID
                        VALUES (
                            1, 1, 'Pretty Great!','I really enjoyed your picture', 01/04/2021
                            )
INSERT INTO Comment (
                            PostId, UserProfileId, Subject, Content, CreateDateTime)
                        OUTPUT INSERTED.ID
                        VALUES (
                            1, 1, 'Glad to be the Second Comment','Looks like the SQL insert is looking', 01/04/2021
                            )

INSERT INTO Comment (
                            PostId, UserProfileId, Subject, Content, CreateDateTime)
                        OUTPUT INSERTED.ID
                        VALUES (
                            2, 1, 'Another good looking post','Test text', 01/04/2021
                            )
